pep-0694.rst

```diff
@@ -39,9 +39,9 @@ Beyond the above, there are a number of major issues with the current API:
   from source.
 
 - It has very limited support for communicating back to the user, limited entirely
-  to the HTTP status code, and status message (something which I'm not sure is
-  even technically valid HTTP?). It has no support for multiple errors, warnings,
-  deprecations, etc.
+  to the HTTP status code and reason phrase, which has been deprecated 
+  since HTTP/2 (:rfc:`7540` #section-8.1.2.4). It has no 
```

Comment:
> I could not find a way to link to a specific section on the referenced RFC...

Here's the magic words. You can change "RFC 7540" to whatever you'd like as the link display text:

```suggestion
  since HTTP/2 (:rfc:`RFC 7540 <7540#section-8.1.2.4>`). It has no 
```

Reply:
In the preview this does not seem to be working. Are you confident this will work on peps.python.org?

Reply:
You have to actually commit the suggestion to regenerate the [preview](https://pep-previews--2685.org.readthedocs.build/pep-0694/), unless you mean you tested it locally? If so, could you provide a link/screenshot, and/or be more specific as to what actual behavior you are seeing?

Unless there's something subtle I'm missing, the syntax is indeed correct.

Reply:
Apologies for the vague phrasing of 'preview'. I was talking about the github preview you can see when editing, which does work with just :rfc:`7540`, but not with this change.
I'll commit this suggested syntax, thanks!

Reply:
It works :heavy_check_mark:

Reply:
> I was talking about the github preview you can see when editing, which does work with just :rfc:`7540`, but not with this change.

Oh, the GitHub preview should not be trusted in the slightest, as its reST parser is rather limited to begin with, and that's without all our custom extensions to process and render the PEPs.